src/renderer/css/components/NewServerModal.scss

```diff
@@ -12,6 +12,10 @@
 
     .Modal_body {
         margin-bottom: 12px;
+        
+        .Modal__body {
```

Comment:
I want to suggest changing the name of one of these, but I have no better suggestions for how to name this. Naming CSS classes is hard 😅